bin/run-bench.js

```diff
@@ -19,9 +18,18 @@ const tests = []
 const globs = []
 const opts = Object.create(null)
 
+// replacement for former async-lib cb
+const testCb = (err, payload) => {
+  if (err) {
+    console.error(err)
+    return
+  }
+  return payload
+}
+
 process.argv.slice(2).forEach(function forEachFileArg(file) {
   if (/^--/.test(file)) {
-    opts[file.substr(2)] = true
+    opts[file.substring(2)] = true
```

Comment:
Curious why this changed, they're semantically doing the same thing?

Reply:
`substr` is a [deprecated method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr), so I went with the alternative that's likely to have a longer support timeline.